TypeScript/README.md

```diff
@@ -119,16 +119,16 @@ var foo = fn();
 
 	// Also bad
 	function performOperation (): {
-		str: string,
-		num: number
+		str: string;
```

Comment:
Why put semicolons here? Was this intentional? I think maybe this is just missing a "var" declaration indead.